src/drivers/hardware.h

```diff
@@ -2,6 +2,19 @@
 
 #include "types.h"
 
-extern u16 monitor, keyboard, clock;
+#define LEM 0x7349F615
+#define KEYBOARD 0x30CF7406
+#define CLOCK 0x12D0B402
 
-void hardwareLoop();
+struct HardwareInfo;
+
+struct HardwareList {
+    u16 size;
+    struct HardwareInfo *hardware;
+};
+
+struct HardwareList hardwareList = {0, 0};
```

Comment:
I wanted to declare a static variable in the source file but I got a .lcomm opcode that isn't easily readable. With this "fix" we get a .zero instead which is easier to parse.